docs/source/noise/column_noise.rst

```diff
@@ -24,3 +24,55 @@ Use a fake name
 
 Make typos
 ----------
+
+Typos occur in survey and administrative datasets when someone -- a survey respondent, a canvasser,
+or someone entering their own information on a form -- types a value incorrectly.
+
+Currently, pseudopeople implements two kinds of typos: inserting extra characters
+directly preceding characters that are adjacent on a keyboard, or replacing a character with one that is adjacent.
+When pseudopeople introduces typos, 10% of them are inserted characters, while the other 90% are replaced characters.
+This is currently not configurable.
+
+To define "adjacent", we use a grid version of a QWERTY keyboard layout
```

Comment:
Should we say somewhere that all adjacent characters are equally likely to be chosen?

Reply:
Yes, good point!